incubator/buzzfeed-sso/templates/auth-deployment.yaml

```diff
@@ -0,0 +1,141 @@
+{{- $fullName := include "buzzfeed-sso.fullname" . -}}
+{{- $googleSecret := .Values.provider.google.customSecret | default (printf "%s-google" ($fullName)) -}}
+{{- $authSecret := .Values.auth.customSecret | default ($fullName) -}}
+{{- $name := include "buzzfeed-sso.name" . -}}
+{{- $authDomain := .Values.auth.domain -}}
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ $fullName }}-auth
+  labels:
+    app: {{ $name }}-auth
+    chart: {{ template "buzzfeed-sso.chart" . }}-auth
```

Comment:
The `app`, `chart` label should be the same across all chart components. How about using `component` label to distinguish them?

Reply:
Good point 👍 . I am using the `container` label now to distinguish them.